larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -637,118 +924,131 @@ void runTest(const TestParam& param) {
-        ::testing::Values(std::array<int, 3>{1, 1, 1},
-                          std::array<int, 3>{3, 3, 1},
-                          std::array<int, 3>{2, 3, 1},
-                          std::array<int, 3>{1, 1, 4},
-                          std::array<int, 3>{3, 3, 4},
-                          std::array<int, 3>{2, 3, 4},
-                          std::array<int, 3>{1, 1, 64},
-                          std::array<int, 3>{3, 3, 64},
-                          std::array<int, 3>{2, 3, 64}),  // filter shape [HWO]
-        ::testing::Values(std::array<int, 2>{1, 1},
-                          std::array<int, 2>{2, 3}),  // strides height/width
-        ::testing::Values(std::array<int, 2>{1, 1},
-                          std::array<int, 2>{3, 2}),  // dilation height/width
-        ::testing::Values(Padding_VALID, Padding_SAME, Padding_ONE),  // padding
-        ::testing::Values(ActivationFunctionType_NONE,
-                          ActivationFunctionType_RELU),  // activation function
-        ::testing::Values(1, 2),                         // number of threads
-        ::testing::ValuesIn(BConv2DOpTest::GetKernelsTuples(*kKernelMap))),
+        Values(std::array<int, 4>{1, 7, 7, 1}, std::array<int, 4>{1, 8, 5, 1},
+               std::array<int, 4>{1, 7, 7, 64}, std::array<int, 4>{1, 8, 5, 64},
+               std::array<int, 4>{1, 7, 7, 130},
+               std::array<int, 4>{1, 8, 5, 130}),  // input shape [BHWI]
+        Values(std::array<int, 3>{1, 1, 1}, std::array<int, 3>{3, 3, 1},
+               std::array<int, 3>{2, 3, 1}, std::array<int, 3>{1, 1, 4},
+               std::array<int, 3>{3, 3, 4}, std::array<int, 3>{2, 3, 4},
+               std::array<int, 3>{1, 1, 64}, std::array<int, 3>{3, 3, 64},
+               std::array<int, 3>{2, 3, 64}),  // filter shape [HWO]
+        Values(std::array<int, 2>{1, 1},
+               std::array<int, 2>{2, 3}),  // strides height/width
+        Values(std::array<int, 2>{1, 1},
+               std::array<int, 2>{3, 2}),  // dilation height/width
+        Values(Padding_VALID, Padding_SAME, Padding_ONE),  // padding
+        Values(ActivationFunctionType_NONE,
+               ActivationFunctionType_RELU),  // activation function
+        Values(1, 2),                         // number of threads
+        ValuesIn(BConv2DOpTest::GetKernelsTuples(*kKernelMap))),
     TestParam::TestNameSuffix);
 
-TEST(BConv2DTests, BConvErrorTest) {
-  // We have to use typedefs or else the template invocation in the type
-  // confuses the pre-processor (EXPECT_DEATH is a macro).
-  typedef BConv2DOpModel<float, float> FP_BConv2DOpModel;
-  typedef BConv2DOpModel<float, std::int8_t> Bitpacked_BConv2DOpModel;
+TEST(BConv2DTests, ReluErrorTest) {
+  LceTensor<float> input_tensor({1, 16, 16, 64});
+  LceTensor<std::int32_t> packed_filter_tensor({128, 3, 3, 64});
+  LceTensor<float> post_tensor({128});
+  LceTensor<float> output_tensor;
+  LceTensor<std::int32_t> packed_output_tensor;
+
```

Comment:
```suggestion

  // We have to use typedefs or else the template invocation in the type
  // confuses the pre-processor (EXPECT_DEATH is a macro).
```

(This adds back a comment that I think is useful to have)

Reply:
Thanks for spotting that, this is an extremely useful comment!